robot_names: 
  - 'husky1'
  - 'husky2'
  - 'husky3'
  - 'husky4'
  - 'spot1'
  - 'spot2'
  - 'spot3'
  - 'spot4'
  - 'aquila1'
  - 'aquila2'
  - 'kaist1'
  - 'zoe1'
  - 'zoe2'
  - 'xmaxx1'
  - 'acl_jackal'
  - 'acl_jackal2'
  - 'sparkal1'
  - 'sparkal2'